Signal/src/view controllers/CountryCodeViewController.m

```diff
@@ -20,17 +24,61 @@ - (void)viewDidLoad {
     [self.navigationController.navigationBar setTranslucent:NO];
     _countryCodes = [self countryCodesForSearchTerm:nil];
     self.title    = NSLocalizedString(@"COUNTRYCODE_SELECT_TITLE", @"");
+    self.searchBar.delegate = self;
 }
 
 - (NSArray *)countryCodesForSearchTerm:(NSString *)searchTerm {
     NSMutableArray *result = [NSMutableArray array];
-    for (NSString *countryCode in [PhoneNumberUtil countryCodesForSearchTerm:nil]) {
+    for (NSString *countryCode in [PhoneNumberUtil countryCodesForSearchTerm:searchTerm]) {
```

Comment:
nit: I feel like it would be easier to test this logic and catch future regressions like this one if the logic were put into the PhoneNumberUtil (or some other non VC class), and unit test at least the happy path .